### PR TITLE
Tin is now in Minetest Game

### PR DESCRIPTION
**Pull request by Thomas--S**

### CODE REVIEW

File: mg.lua [hunk 1]
```diff
@@ -5,20 +5,22 @@ Copyright (c) 2011-2017 Hugo Locurcio and contributors.
...
-	maxheight = 8,
-	seglenghtn = 15,
-	seglenghtdev = 6,
-	segincln = 0,
-	segincldev = 0.6,
-	turnangle = 57,
-	forkturnangle = 57,
-	numperblock = 2
-})
+if not default_tin then
```
Comment (SmallJoker) [inline mg.lua]: Is not a global variable

Comment (Thomas--S) [reply mg.lua]: Thanks!

File: init.lua [hunk 2]
```diff
@@ -10,6 +10,11 @@ Licensed under the zlib license. See LICENSE.md for more information.
 
 moreores = {}
 
+local default_tin = false
+if minetest.registered_items["default:tin_ingot"] then
+	default_tin = true
+end
+
```
Comment (Desour) [inline init.lua:17]: `local default_tin = minetest.registered_items["default:tin_ingot"] ~= nil` would be shorter.

**Comment by Calinou** [comment]
Is there a reason as to why there is no registered alias in case `default:tin_ingot` is registered? This would make converting More Ores tin to minetest_game tin automatic for players who use recent minetest_game versions, and preserve compatibility for stable 0.4.15 players, I guess.

**Comment by Thomas--S** [comment]
@Calinou I think that the alias is here: https://github.com/minetest-mods/moreores/pull/18/files#diff-d7d74284e2e39f6c06d199c8e2bedbd0R336

**Comment by Calinou** [comment]
@Thomas--S Alright, I didn't notice it from a quick look at the diff, thanks. I guess I'll merge this pull request, but I can't right now, I can try this weekend.

Someone with write access could do it too, if they wish and they have tested it first.

**Pull request merged**